.circleci/config.yml

```diff
@@ -61,6 +61,9 @@ jobs:
           command: npm run integration
           environment:
             LOCKBOX_FIREFOX_HEADLESS: "1"
+      - run:
```

Comment:
This *might* be worth running first before any tests, so things fail fast on lint issues

Reply:
That's a very fair point.  Personally, I've cycled between before, after, or parallel.  I think I pushed up after so that we see functional failures before stylistic failures; would be just as content with stylistic first.

@6a68 @meandavejustice do you have a preference?

Reply:
either before or parallel seem optimal to me

Reply:
I would say *not* parallel, because I'd think you'd *want* this to preempt other tests and fail early. But 🤷‍♂️

Reply:
Those are fair points.  What is currently in this branch is a parallel test.  The Travis-CI version ran lint checks in parallel with tests, for that's worth.

If we do go with this, we can mark the `lint` job as required, which would block merging (though an admin could override).

Reply:
I'd probably prefer to run everything in parallel, so we can gather as much diagnostic info as possible from each test run. But that's not a strongly held opinion.